tests/execute/duration/test.sh

```diff
@@ -3,23 +3,46 @@
 
 rlJournalStart
     rlPhaseStartSetup
-        rlRun "tmp=\$(mktemp -d)" 0 "Creating tmp directory"
         rlRun "pushd data"
     rlPhaseEnd
 
-    for method in tmt; do
-        rlPhaseStartTest "Test $method"
-            rlRun "tmt run -vfi $tmp -a execute -h $method test --name good" 0
-            rlRun "tmt run -vfi $tmp -a execute -h $method test --name long" 2
-            rlRun -s "tmt run --last report -fvvv" 2
-            rlAssertGrep "Maximum test time '3s' exceeded." $rlRun_LOG
-            rlAssertGrep "Adjust the test 'duration' attribute" $rlRun_LOG
-            rlAssertGrep "spec/tests.html#duration" $rlRun_LOG
-        rlPhaseEnd
+    for execute_method in tmt; do
+        for provision_method in $PROVISION_METHODS; do
+            rlPhaseStartTest "Test provision $provision_method, execute $execute_method, short tests"
+                rlRun "tmp=\$(mktemp -d)" 0 "Creating tmp directory"
+
+                rlRun -s "tmt run -vfi $tmp -a provision -h $provision_method execute -h $execute_method test --name short" 0
+
+                rlRun "grep 'duration \"5\" exceeded' $tmp/log.txt" 1
+
+                rlRun "rm -rf $tmp" 0 "Removing tmp directory"
+            rlPhaseEnd
+
+            rlPhaseStartTest "Test provision $provision_method, execute $execute_method, long tests"
+                rlRun "tmp=\$(mktemp -d)" 0 "Creating tmp directory"
+
+                rlRun -s "tmt run -vfi $tmp -a provision -h $provision_method execute -h $execute_method test --name long" 2
+                rlAssertNotGrep "00:01:.. errr /test/long/beakerlib (timeout) \[1/2\]" $rlRun_LOG
+                rlAssertNotGrep "00:01:.. errr /test/long/shell (timeout) \[2/2\]" $rlRun_LOG
```

Comment:
Wouldn't it be `00:02:..` if the test script is blocked for `120` seconds?

Reply:
Yes. Seems like a mistake. Fixed in 45e9bd5c654d8c31645fc2e630f2d4ebcfdbd28d.